src/models/config/config.ts

```diff
@@ -38,16 +45,20 @@ export function getConfigPath() {
   return join(getConfigDir(), 'config.json');
 }
 
-/**
- * TODO: Implement
- * Read config from the disk and parse to JSON.
- */
 export async function getConfig(): Promise<ConfigLike> {
-  return await Promise.resolve({
-    name: 'contribution-mate-demo',
-    email: 'demo@contribuion.mate',
-    repository: 'contribution-mate-sync',
-  });
+  try {
+    const data = JSON.parse(await Deno.readTextFile(getConfigPath()));
+    const config = ConfigSchema.parse(data);
+    return config;
+  } catch (err) {
+    if (err instanceof z.ZodError) {
+      const { fieldErrors } = err.flatten();
+      console.log(fieldErrors); // todo: ask what we want to do with it?
+    }
+
+    console.error(err);
+    Deno.exit(1);
```

Comment:
suggestion(blocking): since this is a model, we most likely don't want to decide whether the whole application should exit; we should delegate this job to the caller function instead. Let's remove the error handling and keep only what's inside the `try` block. The caller function will decide how the program will recover or exit.